src/Aspire.Hosting/Dashboard/DcpDataSource.cs

```diff
@@ -86,6 +91,24 @@ await foreach (var (eventType, resource) in _kubernetesService.WatchAsync<T>(can
         }
     }
 
+    private static bool IsFilteredResource<T>(T resource) where T: CustomResource
+    {
+        // We filter out any resources that start with aspire-dashboard (there are services as well as executables).
+        if (resource.Metadata.Name.StartsWith("aspire-dashboard", StringComparisons.ResourceName))
+        {
+            if (Environment.GetEnvironmentVariable("DOTNET_ASPIRE_SHOW_DASHBOARD_RESOURCES") is { } showDashboardResourcesValue)
+            {
+                return bool.TryParse(showDashboardResourcesValue, out var parsedShowDashboardResourcesValue) && !parsedShowDashboardResourcesValue;
```

Comment:
Some VS env vars use `1` and `0`. Is there precedent on only accepting valid bool strings?